packages/front-end/components/Experiment/TabbedPage/AnalysisSettings.tsx

```diff
@@ -150,26 +151,46 @@ export default function AnalysisSettings({
 
         {!isPublic && (
           <div className="row">
-            <div className="col-4">
+            <div className="col-4 mb-4">
               <div className="h5">Data Source</div>
               <div>{datasource ? datasource.name : <em>none</em>}</div>
             </div>
 
-            <div className="col-4">
+            <div className="col-4 mb-4">
               <div className="h5">Experiment Assignment Table</div>
               <div>
                 {assignmentQuery ? assignmentQuery.name : <em>none</em>}
               </div>
             </div>
 
+            {experiment.activationMetric && (
+              <div className="col-4 mb-4">
+                <div className="h5">Activation Metric</div>
+                <div>
+                  {getExperimentMetricById(experiment.activationMetric)?.name}
+                </div>
+              </div>
+            )}
+
+            <div className="col-4 mb-4">
+              <div className="h5">Segments</div>
```

Comment:
This should be singular since you can only select one.  Also, we don't really want people using segments for experiments, so maybe only show this if a segment is set to avoid drawing too much attention to it.

Reply:
fixed